contrib/build_executable.jl

```diff
@@ -219,12 +236,30 @@ function emit_cmain(cfile, exename, relocation)
 end
 
 function emit_userimgjl(userimgjl, script_file)
-    f = open(userimgjl, "w")
-    write( f, """
-        include(\"$(script_file)\")
-        """
-    )
-    close(f)
+    open(userimgjl, "w") do f
+        write( f, "include(\"$(escape_string(script_file))\")")
+    end
+end
+
+function find_system_gcc()
+    # On Windows, check to see if WinRPM is installed, and if so, see if gcc is installed
+    @windows_only try
+        require("WinRPM")
+        winrpmgcc = joinpath(WinRPM.installdir,"usr","$(Sys.ARCH)-w64-mingw32",
+            "sys-root","mingw","bin","gcc.exe")
+        if success(`$winrpmgcc --version`)
+            return winrpmgcc
+        end
+    end
+
+    # See if `gcc` exists
+    try
+        if success(@windows ? `cmd /c gcc -v` : `gcc -v`)
```

Comment:
So this `try` block checking whichever `gcc` is on the path should be `@unix_only`.